script.js

```diff
@@ -20,3 +95,81 @@ document.querySelector('.menu').addEventListener('click', () => {
     closeButton.remove();
   });
 });
+
+function createpopups() {
+  const button = document.querySelectorAll('.card .button');
+  const buttonArr = Array.from(button);
+  buttonArr.map((el, i) => {
+    el.addEventListener('click', () => {
+      const popupContainer = document.createElement('div');
+      popupContainer.classList.add('popup-container');
+
+      const popup = document.createElement('div');
+      popup.classList.add('popup');
+
+      popupContainer.appendChild(popup);
```

Comment:
Hello @team, this popup looks great because you have done a good job 👍🏻! However, the overall look of it does not match that in the figma design. I suggest that you make the required changes to make look closer to the design. See in the comparison below 👇🏻.

| Your popup | Figma design popup |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/4247740/151232777-79747101-0051-4dad-82db-6bbe10493fb7.png) | ![image](https://user-images.githubusercontent.com/4247740/151232855-f948b7f1-2174-48a2-87db-1fcc605eecfb.png) |

- [ ] You could try to decrease the width of the popup so that its not full width on the desktop view. What do you think?
- [ ] Align the project title and the buttons well in reference to the figma design